modules/ocf_mirrors/files/healthcheck

```diff
@@ -19,9 +20,33 @@ import requests
 Mirror = namedtuple('Mirror', ['url', 'updated_at'])
 
 
-def get_updated(mirror_url):
-    """Find the time the host was last updated.
+def update_func(healthcheck):
+    if healthcheck == 'debian':
+        return get_updated_debian
+    elif healthcheck == 'unix_timestamp':
+        return get_updated_ts
+    elif healthcheck == 'datetime':
+        return get_updated_datetime
+    else:
+        raise ValueError('Unsupported type: {}'.format(healthcheck))
+
+
+def get_updated_datetime(mirror_url):
+    """Find the time the host was last synced"""
+    req = requests.get(mirror_url)
+    req.raise_for_status()
+    return dateutil.parser.parse(req.text.splitlines()[0])
+
 
+def get_updated_ts(mirror_url):
+    """Find the time the host was last synced"""
```

Comment:
Same as the comment above, I think this description should be different from that one and have more useful information.